bin/report_form_registration_problems.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+from __future__ import annotations
+
+import sys
+from collections.abc import Sequence
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import django
+
+import click
+from tabulate import tabulate
+
+if TYPE_CHECKING:
+    from openforms.forms.models import FormRegistrationBackend
+
+SRC_DIR = Path(__file__).parent.parent / "src"
+sys.path.insert(0, str(SRC_DIR.resolve()))
+
+
+def check_registration_backend(registration: FormRegistrationBackend) -> str | None:
+    from openforms.registrations.contrib.objects_api.config import VersionChoices
+
+    match (registration.backend, registration.options):
+        case ("objects_api", {"version": int() as version}) if version == VersionChoices.V2:
```

Comment:
```suggestion
        case ("objects_api", {"version": 2}):
```
this is equivalent

However, what it doesn't capture is that the `version` key may be absent, and that (currently) defaults to it being v2. I don't know if we should consider that case or not.

Reply:
Check, yeah i thought: 'we have a constant for this, why not use it :)'

Yeah checking for no version also crossed my mind, for the report and for the fix script. But from the moment we started working with versions, it defaulted to version 1 (which was recently (start of 2025) changed to version 2).
The import does handle this, so this case would be for Open Forms environments upgrading from 2.x to 3.1? But i assume some migration stuff is in place..

Maybe it would be best to play this save... I'll add it to the report and fix scripts

Reply:
> we have a constant for this, why not use it :)

sure, fine - my remark is about the `"version": int() as version if version == 2` part :P